models/friend-model.js

```diff
@@ -1,9 +1,7 @@
 const { Schema, SchemaTypes } = require('mongoose');
 
 const FriendSchema = new Schema({
-  username: { type: String, required: true },
-  friendSince: { type: Date, default: () => new Date()},
-  additionalInfo: { type: SchemaTypes.ObjectId, ref: "User" }
-})
+  addresseeId: { type: SchemaTypes.ObjectId, ref: "User" }
```

Comment:
but it works only on 1 way
I am Eugene
I have friend 'Jony'
But Jony haven't friend 'Eugene'

Reply:
But when one user accepts friend request I'm modifying both users. Or I don't understand something

Reply:
I see.

Okay, let it be as it is now, It works)

The better way is not modify users, it can be in the friend model.

https://dba.stackexchange.com/questions/135941/designing-a-friendships-database-structure-should-i-use-a-multivalued-column